### DISCUSSION TITLE
Page recherche use case MC

### DISCUSSION THREAD

**Discussion by DaFrenchFrog** [etalab/catalogage-donnees#212] [Ideas]
@Volubyl  @johanricher @florimondmanca 
Je lance la discussion ici parce que la demande n'est pas un must have mais je pense que cela rendrait l'expérience plus fluide et plus cohérente. Je ne sais si cette évolution demande un travail conséquent mais je me dis que cela pourrait être envisageable pour fin mai.

## Contexte
Lorsque l'on arrive sur la page d'accueil la liste des cartes est affichée et il y a un bloc recherche permettant de saisir des termes, et un clic amène sur la page recherche affichant les résultats. On peut également cliquer sur le menu rechercher qui affiche lui aussi un bloc recherche de style différent et permet également d'afficher les résultats.

## Problème
Etant donné que nous avons très peu de fonctionnalités, la page d'accueil et la page de recherche sont très similaires. Si nous affichions directement l'ensemble des cartes par défaut dans la page recherche nous aurions même un affichage quasiment identique. La seule différence réside dans le fait que nous affichons un extrait du descriptif dans les résultat, alors que ce n'est pas le cas sur la home.

Lorsque l'on saisit un terme sur l'accueil on se retrouve sur une page quasiment identique si ce n'est que le bloc de recherche est légèrement différent et qu'un extrait de descriptif est affiché. Cela peut perturber l'expérience sans pour autant apporter de bénéfice.

## Proposition
Je propose simplement de supprimer la page recherche dont le rôle est finalement bien rempli sur l'accueil. Lorsque l'on saisira une recherche on pourra directement afficher les résultat à la place de la liste "par défaut" et si le terme est trouvé dans le descriptif on pourra l'afficher de la même façon que sur les résultats actuels.

Pour résumer :
- Supprimer la rubrique "Rechercher" du menu
- Sur l'accueil, Renommer "Dernières contribution" en "XXX fiches de données"
- Lors d'une recherche n'afficher le descriptif sur les cartes que lorsque les termes y ont été trouvés

Je pense que cela simplifiera grandement l'outil dans son usage et rendra l'ensemble plus cohérent. Qu'en pensez-vous ?

**Comment by florimondmanca** [comment]
@Volubyl @DaFrenchFrog 

Grâce à https://github.com/etalab/catalogage-donnees/pull/223, qui a uniformisé le bloc de recherche sur la page d'accueil et la page de recherche, pour ma part il me semble pertinent de se débarrasser de la page de recherche et basculer sa fonctionnalité sur la page d'accueil.

Si on souhaite le faire on peut créer une issue avec le bouton "Create issue from discussion" à droite de cette page.

**Reply by Volubyl** [reply]
on va p-e attendre d'avoir l'avis de @johanricher avant non ?

**Comment by Volubyl** [comment]
je me faisait la réflexion en bossant sur ces pages. Elle me donnaient un peu l'impression de faire doublon

**Comment by johanricher** [comment]
Actuellement on a une page "recherche" qui est designée : un champ plein texte, et des résultats qui s'affichent en dessous. Un jour cette page sera enrichie d'autres fonctionnalités (par exemple celles liées à #126)  

On n'a pas à proprement parler de page "home" designée comme telle, c'est à dire selon mon de vue avec du contenu éditorialisé qui permet de comprendre le service et à quoi il sert quand on arrive dessus. Peut-être une headline, un petit visuel ? Peut-être une liste de quelques jeux de données, par exemple les derniers ajoutés ? (un jour peut-être ceux avec le plus de contributions mais pour l'instant on n'a pas ces métadonnées mais c'est dans cet esprit que je vois les choses). Cela correspond à ce qui devrait être selon moi notre page d'accueil, mais cela se discute dans le contexte du projet. Je suis en tout cas favorable à ce qu'on s'inspire le plus possible de data.gouv.fr car notre public est le leur et donc ça fluidifie l'embarquement des utilisateurs sur quelque chose qui leur est déjà familier.

Actuellement donc on n'a pas de page d'accueil à proprement parler, a-t-on besoin d'une ? C'est la première chose à vérifier. Si c'est le cas il faut répondre à ce besoin, faire du design et l'inscrire dans la feuille de route en termes de priorités.

Dans l'immédiat, comme on n'a pas de page "Accueil" je suis favorable à ce que la page "Recherche" la remplace et prenne le nom de "Accueil" pour n'avoir qu'une seule page/onglet au lieu de deux.

Est-ce que j'ai bien compris la problématique et ça rejoint ce que vous dites ?

**Reply by DaFrenchFrog** [reply]
C'est tout à fait cela si ce n'est que je le formulerais plutôt en "on supprime la page recherche" puisque la page d'accueil fonctionne très bien telle quelle (c'est àà dire en affichant directement les résultat sans attendre qu'un texte de recherche soit saisi).

A terme on aura sûrement des choses à "dire" lorsqu'un utilisateur se connecte sur l'outil, mais pour l'instant ce n'est pas une priorité et jusqu'ici tous les tests que nous avons fait ne montrent pas de besoin particulier d'avoir autre chose que ce que nous avons actuellement.
